pulseguardian/model/pulse_user.py

```diff
@@ -27,15 +27,18 @@ class PulseUser(Base):
         Queue, backref='owner', cascade='save-update, merge, delete')
 
     @staticmethod
-    def new_user(username, password='', owner=None, create_rabbitmq_user=True):
+    def new_user(username, password='', owners=None, create_rabbitmq_user=True):
         """Initializes a new user, generating a salt and encrypting
         his password. Then creates a RabbitMQ user if needed and sets
         permissions.
 
         :param create_rabbitmq_user: Whether to add this user to the rabbitmq
         server via the management plugin.  Used by tests.
         """
-        pulse_user = PulseUser(owner=owner, username=username)
+        # ensure ``owners`` is a list
```

Comment:
In general, comments should start with capitals and end with appropriate punctuation.